content/en/serverless/installation/dotnet.md

```diff
@@ -149,30 +149,44 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 {{% /tab %}}
 {{% tab "Custom" %}}
 
-1. Install the Datadog Lambda Extension
+1. Install the Datadog Tracer
 
     [Configure the layers][1] for your Lambda function using the ARN in the following format:
 
-    `arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:{{< latest-lambda-layer-version layer="extension" >}}`
+    ```sh
+    # Use this format for x86-based Lambda deployed in AWS commercial regions
+    arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-dotnet:{{< latest-lambda-layer-version layer="dd-trace-dotnet" >}}
 
-2. Install the Datadog .NET APM client
+    # Use this format for arm64-based Lambda deployed in AWS commercial regions
+    arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-dd-trace-dotnet-ARM:{{< latest-lambda-layer-version layer="dd-trace-dotnet" >}}
```

Comment:
You can now remove the banner at [line 16](https://github.com/DataDog/documentation/pull/14785/files#diff-7687466f997626a211de4bf0497bbf58422c43337cf19bf39f861cc238ae9edbR16).

Reply:
Fixed!